mu-plugins/central-hub/tests/phpunit/fixtures/meta-box-members-runtime-config.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+return [
+	'meta_box.members' => [
+		'add_meta_box'  => [
+			'title'         => 'Tour Member Profile Information',
+			'screen'        => [ 'members' ],
+			'context'       => 'normal',
+			'priority'      => 'default',
+			'callback_args' => null,
+		],
+		'custom_fields' => [
+			'residence_city'  => [],
+			'residence_state' => [],
+			'role'            => [
+				'is_single'    => true,
+				'default'      => '',
+				'delete_state' => '',
+				'sanitize'     => 'sanitize_text_field',
+			],
+			'is_active' => [
+				'default'      => 0,
+				'delete_state' => 0,
+				'sanitize'     => 'int_val',
+			]
+		],
```

Comment:
@rgadon107 Notice that I changed the custom fields' configuration parameters and added a new one. 

Why? I did this to ensure that the initialized custom fields' configuration that gets stored in config store is as expected. 

What do I mean? `autoloadConfigurations()` does multiple things:

1. It uses `loadConfigFromFilesystem()` to pull the config file from the filesystem and then load it into the Config Store.
2. It uses `init_custom_fields_configuration()` to merge the default custom fields' configuration parameters and structure with the original config to ensure that the final custom fields' config has all of the parameters it needs.

To test that this works, we want a fixture (original config) that only has the custom parameters and not the default ones. 

Does that make sense?

You might be wondering why we do this. Right? To save code. The config files in each plugin are only required to specify the custom parameters. What does that mean? To save code, you only need to specify what's different. For example, you may have a checkbox (such as `is_active`) where it's configuration is different than a text field.

Does that make sense?

Reply:
@hellofromtonya Yes.

Reply:
@hellofromtonya Yes.

Reply:
@hellofromtonya Yes.